### ISSUE TITLE
cabal.project refers to things on local filesystem

### ISSUE THREAD

**Issue by chessai** [rrruko/kafka-client#4]
is this necessary?

```
[chessai@chessai-kudu:~/development/kafka-client]$ cat cabal.project
packages: . ../sockets ../castagnoli

source-repository-package
  type: git
  location: https://github.com/haskell-primitive/primitive-slice
  tag: 554f1f0cf00d0c445b4210ce8367be5269e6ce64

source-repository-package
  type: git
  location: https://github.com/andrewthad/posix-api
  tag: 6344f841b969cb70195e49093be1504b4a84f7c5
```

**Comment by rrruko** [comment]
I think those local filesystem dependencies can be pulled from github instead. I'll fix that.

**Comment by rrruko** [comment]
Fixed by ca516ec.
